### PR TITLE
fix ticket:3941, disable tests Modelica_Synchronous.WorkInProgress.*

### PR DESCRIPTION
**Pull request by adrpo**

### CODE REVIEW

**Comment by adrpo** [comment]
Ran a library test here to see if this change is working:
https://dev.openmodelica.org/libraries/Modelica_Synchronous/BuildModelRecursive.html

File: Examples/BuildModelRecursive.mos [hunk 1]
```diff
@@ -81,7 +81,9 @@ loadModelCommand:="\n"+customCommands+"\nloadModel("+libraryString+",{\""+librar
 omc:=getInstallationDirectoryPath()+"/bin/omc";
 dygraphs:=getInstallationDirectoryPath()+"/share/doc/omc/testmodels/dygraph-combined.js";
 
-a:={typeNameString(x) for x guard isExperiment(x) in getClassNames(library,recursive=true,sort=sortFiles)};
+a:={typeNameString(x)
+    for x guard isExperiment(x) and not regexBool(typeNameString(x), "^Modelica_Synchronous\.WorkInProgress")
```
Comment (sjoelund) [inline Examples/BuildModelRecursive.mos]: Isn't this a lexer error in Modelica? Should be `"\\."` and not `"\."`

Comment (adrpo) [inline Examples/BuildModelRecursive.mos]: @sjoelund: i guess, so, i'll kill the pull request!

**Comment by OpenModelica-Hudson** [comment]
The tests run correctly according to [OpenModelica_TEST_PULL_REQUEST 2016-05-31_07-42-03](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/2855/console).

**Pull request merged**